tests/forms_tests/tests/test_forms.py

```diff
@@ -867,7 +871,17 @@ class SongForm(Form):
         f = SongForm(data)
         self.assertEqual(f.errors, {})
 
+        # checking whether MultiValueDictLike.getlist() is called,
+        # even if composers value is a string
+        data = MultiValueDictLike(dict(name='Yesterday', composers='J'))
```

Comment:
I'm not sure why the existing test uses `dict(...)` -- let's use `{'k': 'v'}` for the new tests.